src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -636,19 +623,22 @@ private static int GetPreambleLength(byte[] data, int dataLength, Encoding encod
             }
```

Comment:
Good catch. Fixed. (Which also highlights we separately need some tests of various encodings.)